src/fastmcp/tools/tool.py

```diff
@@ -142,6 +147,7 @@ def __eq__(self, other: object) -> bool:
 def _convert_to_content(
     result: Any,
     _process_as_single_item: bool = False,
+    _serializer: Callable[[Any], str] | None = None,
```

Comment:
This could be `serializer` (no leading underscore) and appear before `_process_as_single_item` in the kwargs order -- the reason `_process_as_single_item` has the underscore is because its only meant to be called in recursive calls to `_convert_to_content` and is essentially a "private" implementation detail of the function itself.